WORKSPACE.bazel

```diff
@@ -81,8 +84,20 @@ llvm_toolchain(
     # https://github.com/grailbio/bazel-toolchain/blob/ceeedcc4464322e05fe5b8df3749cc02273ee083/toolchain/cc_toolchain_config.bzl#L148-L150
     link_libs = {
         "": ["--driver-mode=g++"],
```

Comment:
unrelated, but you should switch to `-fsanitize-link-c++-runtime`, as we found some issues with `--driver-mode=g++`. It also is much more obvious what the flag is supposed to do.

Reply:
Probably will do in a follow up.